prisma/schema.prisma

```diff
@@ -26,16 +26,17 @@ model Specialist {
   lastName          String           @db.VarChar(64)
   surname           String?          @db.VarChar(64)
   specializations   Specialization[]
-  gender            Gender
+  gender            Gender?
   yearsOfExperience Int?             @db.SmallInt
-  formatOfWork      FormatOfWork
+  formatOfWork      FormatOfWork?
   placesOfWork      PlaceOfWork[]
   therapies         Therapy[]
-  isFreeReception   Boolean
-  description       String           @db.Text
+  isFreeReception   Boolean?
```

Comment:
🟠 Why gender, format of work, is free reception are optional fields?

Reply:
There should be an option to create a specialist in draft mode which would have at least firstName, lastName and therapies selected. That's why all other fields are optional.